javaslang/src/test/java/javaslang/collection/VectorPropertyTest.java

```diff
@@ -29,6 +29,20 @@ public void shouldCreateAndGet() {
             for (int j = 0; j < actual.size(); j++) {
                 assertThat(expected.get(j)).isEqualTo(actual.get(j));
             }
+
+            final Vector<Integer> actualInt = Vector.ofAll(LeafType.<int[]> asPrimitives(int.class, expected));
```

Comment:
create primitive int[] array and test that it contains the correct elements and that it's stored as `int[]`, not `Object[]`